Yvand.EntraCP/Yvand.EntraClaimsProvider/EntraIDEntityProvider.cs

```diff
@@ -180,7 +190,7 @@ protected virtual void BuildFilter(OperationContext currentContext, List<EntraID
 
             List<string> userFilterBuilder = new List<string>();
             List<string> groupFilterBuilder = new List<string>();
-            List<string> userSelectBuilder = new List<string> { "UserType", "Mail" };    // UserType and Mail are always needed to deal with Guest users
+            List<string> userSelectBuilder = new List<string> { "Id", "UserType", "Mail" };    // UserType and Mail are always needed to deal with Guest users
```

Comment:
And `Id` was added because of the groups?

Reply:
Yes, basically, now I declare that I always want to get the user ID of each user returned by Graph